railz-ai/railz-uikit
# chore: refactor entire project, improve DX

This PR:
- Greatly simplifies the development and build flow for the project:
  - Removes (efectively archiving) the unused Vue and Angular Stencil Wrappers around railz-ui-kit
  - Removes the unused (and uncoded, because it only had some examples) railz-icons
  - Puts every package inside a single packages folder, including the docs package
  - It removes the codesandbox scripts, simplifying the script management
  - It updates several packages, most importantly Lerna and the linting utilities.
  - Provides useful scripts to test and develop locally (check the README.md changes) with a much faster flow.
- Fixes the docs package, so that it works locally and in the remote (deployed). This means that it can be used to test changes locally, without relying on codesandbox.
- Updates tests to have coverage in the latest node@20 (current LTS)

To test this:
- I would deleting deleting the clone you have currently of this repo, and cloning it again because I found that there are numerous cached (and git-ignored) files that can bring unintended changes and errors.
- Read the README, and follow the development instructions.
- Check if you can develop locally using the dev:docs and dev:components scripts
- Build the packages
